modules/carousel/jetpack-carousel.js

```diff
@@ -445,6 +445,11 @@ jQuery(document).ready(function($) {
 				return;
 			}
 
+			// skip if the parent is not a link
```

Comment:
Looks good, but one tiny nitpick. We're not talking about the parent here, but about the container, right?